EEBOS/SCROWL
# Chore/scorm api [sc-664]

### Short summary

- restructured the scorm packaging, making it modular and placing it into its own service
- added a basic implementation of a scorm runtime handler that connects to the API provided by the LMS
- added a number of utility files from owl ui

---

### Test steps

1. launch the application
1. click 'Export'
1. a scorm packaged `.zip` file will appear at `apps/course-authoring/src/main/services/exporter/course-packs`
    1. you'll see creation logs in the terminal
    1. you'll see a `packaged Done` message in the dev tools console
1. Create a new course in the LMS and Upload the `.zip`
1. Publish the course
1. Assign your user to the course
1. Start the course
1. Click `Lesson State` 
    1. You should see an alert stating the lesson is incomplete
1. Click `Mark Lesson Complete`
1. Exit the course and re-enter
1. Click `Lesson State`
    1. You should see an alert stating the lesson is complete
